los_keeper/include/los_keeper/type_manager/type_manager.h

```diff
@@ -3,13 +3,47 @@
 //
 #ifndef LOS_KEEPER_TYPE_MANAGER_H
 #define LOS_KEEPER_TYPE_MANAGER_H
+
 #include "los_keeper/bernstein_polynomial_utils/bernstein_polynomial_utils.h"
 #include "thread"
 #include <pcl/point_cloud.h>
 #include <pcl/point_types.h>
+
 namespace los_keeper {
 typedef pcl::PointXYZ PclPoint;
 typedef pcl::PointCloud<PclPoint> PclPointCloud;
+
+struct PredictionParam {
+  int num_sample;                // The number of primitives
```

Comment:
Can you modify them to have a default value?
int num_sample{0} for all params!

Reply:
Okay!